From 977af616e53f21df97668ca4b76ceb20737df9f3 Mon Sep 17 00:00:00 2001
From: nl6720 <nl6720@gmail.com>
Date: Tue, 5 May 2020 12:21:55 +0300
Subject: [PATCH] [kodi-standalone.sh] Replace which with command -v

The which utility is not guaranteed to be installed. Use the portable shell builtin command -v.
---
 tools/Linux/kodi-standalone.sh.pulse | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tools/Linux/kodi-standalone.sh.pulse b/tools/Linux/kodi-standalone.sh.pulse
index c4d556434427..5273094b5574 100644
--- a/tools/Linux/kodi-standalone.sh.pulse
+++ b/tools/Linux/kodi-standalone.sh.pulse
@@ -1,8 +1,8 @@
-PULSE_START="$(which start-pulseaudio-x11)"
+PULSE_START="$(command -v start-pulseaudio-x11)"
 if [ -n "$PULSE_START" ]; then
   $PULSE_START
 else
-  PULSE_SESSION="$(which pulse-session)"
+  PULSE_SESSION="$(command -v pulse-session)"
   if [ -n "$PULSE_SESSION" ]; then
     XBMC="$PULSE_SESSION $XBMC"
   fi
